edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Docker/Microsoft.Azure.Devices.Edge.Agent.Docker.csproj

```diff
@@ -10,7 +10,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="5.0.0" />
-    <PackageReference Include="Docker.DotNet" Version="3.125.2" />
+    <PackageReference Include="Docker.DotNet" Version="3.125.12" />
```

Comment:
Do we need to update Docker.DotNet here? In fact, so we still need Microsoft.Azure.Devices.Edge.Agent.Docker?

Reply:
Earlier I tried removing the dependency altogether. But there are still things that use it in Edge.Agent.Docker (which is still needed because components are depended on from things in AgentModule / EdgeletModule / etc.). I made a pass through Edge.Agent.Docker though to get rid of all not-needed components.

For the dependency version, I went back and forth about it. On one hand leaving it to what it was will minimize changes and tests still pass. But on the other hand we know it isn't compat with new docker versions which makes me suspect. My intuition is that bumping it is the safe bet since tests still pass.

Reply:
Makes sense.